platforms/corda-ent/charts/idman/values.yaml

```diff
@@ -71,84 +61,117 @@ vault:
   role:
   # Provide the authpath configured to be used.
   # Eg. authpath: entcordacenm
-  authpath:
+  authPath:
   # Provide the service account name autheticated to vault.
   # NOTE: Make sure that the service account is already created and autheticated to use the vault.
   # Eg. serviceaccountname: vault-auth
-  serviceaccountname:
-  # Provide the vault path where the  certificates are stored
+  serviceAccountName:
+  # Provide the vault path where the certificates are stored
   # Eg. certsecretprefix: secret/cenm-org-name
-  certsecretprefix:
+  certSecretPrefix:
  
+#############################################################
+#                    Idman Configuration                    #
+#############################################################
 
-# Provide volume related specifications
-volume:
- # Ex baseDir: /opt/corda
-  baseDir: 
-
-# idman service 
+# Idman 'main' service
 service:
-  # Ex. port: 10000
+  # Eg. port: 10000
   port: 
```

Comment:
all ports external port, internal port etc should be under a single service key.

Reply:
Fixed in [ae9a768](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/992/commits/ae9a76899ddcce9ea9e2067c3ba79ebc784c08c5); services have been grouped under common key (`services`).